doc/sphinx/language/cic.rst

```diff
@@ -1,4 +1,4 @@
-Typing rules
+CIC: Typing rules
```

Comment:
I changed it because I had trouble remembering where to find the CIC info.  CIC is the underpinning of Coq.  IIUC this is the chapter that describes it.  (First sentence: "The underlying formal language of Coq is a Calculus of Inductive Constructions (CIC) whose inference rules are presented in this chapter.")

What sections of "Core language" are part of CIC?

Reply:
Technically, all the "Core language" sections are describing CIC (with as little elaboration as possible) and the "Typing rules" section are only about the typing inference rules (in particular, the conversion rules are also part of CIC but not mentioned there). So, perhaps, it's this introduction that needs to be rewritten.

Reply:
OK, I'll drop the title change.  Later on maybe the "Core language" section should be "Core language: CIC" or just "CIC language"; "core" is not very descriptive.